### ISSUE TITLE
If `NavigationAppBar` is null in `NavigationView` then the indicator is misaligned

### ISSUE THREAD

**Issue by mzdm** [bdlukaa/fluent_ui#41]
`fluent_ui: ^2.0.2`

```dart
NavigationView(
      useAcrylic: false,
      // Uncomment this and indicator works fine.
      // appBar: NavigationAppBar(),
      content: NavigationBody(
        index: selectedIndex,
        children: [
          InputPage(),
          SettingsPage(),
          SettingsPage(),
        ],
      ),
      pane: NavigationPane(
        displayMode: PaneDisplayMode.compact,
        selected: selectedIndex,
        onChanged: (i) => setState(() => selectedIndex = i),
        items: [
          PaneItem(
            icon: const Icon(Icons.input),
            title: 'Inputs',
          ),
          PaneItem(
            icon: const Icon(Icons.format_align_center),
            title: 'Forms',
          ),
        ],
        footerItems: [
          PaneItemSeparator(),
          PaneItem(
            icon: const Icon(Icons.settings),
            title: 'About app',
          ),
        ],
      ),
    );
```

![image](https://user-images.githubusercontent.com/67197047/119988820-c9807180-bfc6-11eb-804f-862f9ce68dd1.png)

**Comment by bdlukaa** [comment]
Reproducible on the latest master

**Comment by mzdm** [comment]
> Published version 2.0.3 that fixes this issue

Can confirm it's working now properly, thanks for the fast fix! 💙
